quickstarts/redis-prometheus/config.yml

```diff
@@ -53,19 +54,25 @@ keywords:
   - remote
   - write
   - kubernetes
+  - exporter
   - database
 
 # Reference to install plans located under /install directory
 # Allows us to construct reusable "install plans" and just use their ID in the quickstart config
 installPlans:
   - setup-prometheus
+  - setup-prometheus-agent-mode
```

Comment:
This install plan does not exist. Please use an existing one or create a new one!

Reply:
pending in https://github.com/newrelic/newrelic-quickstarts/pull/1503